setup.py

```diff
@@ -24,6 +24,23 @@
     long_description_content_type="text/markdown",
     url="https://github.com/SymbiFlow/prjxray",
     packages=['prjxray'],
+    install_requires=[
+        'sdf-timing @ git+https://github.com/symbiflow/python-sdf-timing',
+        'fasm @ git+https://github.com/symbiflow/fasm',
+        'intervaltree',
+        'numpy',
+        'openpyxl',
+        'ordered-set',
+        'parse',
+        'progressbar2',
+        'pyjson5',
+        'pytest',
+        'pyyaml',
+        'scipy>=1.2.1',
+        'simplejson',
+        'sympy',
+        'textx',
```

Comment:
Are all these dependencies *actually* needed? These seem like the dependencies for the whole prjxray fuzzer tooling not the prjxray python library?

Reply:
Poke, this comment is valid, and should be fixed.

Reply:
Should be fixed now. The only dependencies added are ones that are needed by scripts in the prjxray directory scripts